c7n/resources/servicecatalog.py

```diff
@@ -25,6 +25,7 @@ class resource_type(TypeInfo):
         enum_spec = ('list_portfolios', 'PortfolioDetails', None)
         detail_spec = ('describe_portfolio', 'Id', 'Id', None)
         arn = 'ARN'
+        arn_type = 'portfolio'
```

Comment:
note this one is an addition, where as the other type modifications in this pr are fixes.